File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
		<!--分页查询所有--> 
		this.findPage=function(page, rows){
			return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
		}
		this.search=function(page, rows,searchEntity){
			return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
		}

		<!--保存方法--> 
		this.add=function(entity){
		return $http.post('../brand/add.do',entity);
			
		}
		this.update=function(entity){
			return $http.post('../brand/update.do',entity);
		}
		<!--查询一个-->
		this.findOne=function(id){
			return $http.get('../brand/findOne.do?id='+id);
		}
		
		<!--删除-->
		this.dele=function(selectIds){
			return $http.get('../brand/delete.do?ids='+selectIds);
		}
		this.selectOptionList=function(){
			return $http.get('../brand/selectOptionList.do');
		}
	})

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;


import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;
import entity.Result;



public interface BrandService {
	public List<TbBrand> findAll();
	
	//品牌分页查询
	public PageResult findPage(int pageNum,int pageSize);
	
	//品牌增加
	public void add(TbBrand tbBrand);
	//品牌修改 先查询
	public TbBrand findOne(long id);
	//品牌修改 修改数据库
	public  void update(TbBrand tbBrand);
	//删除
	public void delete(long[] ids);
	//条件查询
	public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);
	/**
	 * select2 查找品牌
	 * @return
	 */
	public List<Map> selectOptionList();
	
}
